src/makeCalculator.js

```diff
@@ -5,6 +5,39 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, n) {
+      this.result = operation(this.result, n);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(a, b) {
+      return a + b;
```

Comment:
[CODE STYLE] - Instead of returning the sum directly, you should use the assignment operator `+=` to add to the existing value. This applies to the `add`, `subtract`, `multiply`, and `divide` methods.